src/docs/product/data-management-settings/server-side-sampling/index.mdx

```diff
@@ -0,0 +1,117 @@
+---
+title: Server-side Sampling
+sidebar_order: 1
+description: "Learn about how you can control which transactions Sentry retains."
+---
+
+<Note>
+
+This feature is in beta and is only available if your organization is participating in its limited release. Features in beta are still in-progress and may have bugs. We recognize the irony. If you’re interested in participating, [join the waitlist](https://sentry.io/for/performance/#updates-signup).
+
+</Note>
+
+Capturing a single trace involves minimal overhead, but capturing traces for every page load or every API request may add an undesirable load to your system. Sampling your events allows you to better manage the number of events sent to Sentry, so you can tailor your volume to your organization's needs. 
+
+_Server-side sampling_ lets you control which transactions Sentry retains by setting sampling rules and rates, so you see more of the transactions you want to explore further, and less of the ones you don’t. In contrast to [client-side sampling](/platform-redirect/?next=/configuration/sampling/), server-side sampling allows you to manage your sampling configuration centrally and lets you make changes without the need to reconfigure the SDK and redeploy your application.
+
+![The Server-side Sampling page with a list of rules](server-side-sampling.png)
+
```

Comment:
```suggestion

> [A trace](/product/sentry-basics/tracing/) A trace represents the record of the entire operation you want to measure or track, like page load, an instance of a user completing some action in your application, or a cron job in your backend. A trace consists of one or more transactions and it can be distributed across multiple services or projects.

,  

```